### PR TITLE
Saithat/h4 ish 19 phone link

### PR DESCRIPTION
**Pull request by saithat**

### CODE REVIEW

File: src/helpers/PhoneLink.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns what the input is (zip code, county name, or address)
+ * @Param str The string to convert
+ * @Returns A string telling what the input is
```
Comment (VishalAiely) [inline src/helpers/PhoneLink.ts]: I think these comments are incorrect

**Review by VishalAiely — DISMISSED** [review]
Should fix the comment on the function but the code looks really good, thanks for also including tests for validation 👍

**Review by huncfut — DISMISSED** [review]
Seems good!

File: src/helpers/PhoneLink.ts (hunk 1 above)
Comment (saithat) [reply src/helpers/PhoneLink.ts]: changed!

**Review by VishalAiely — APPROVED** [review]
Thanks for the fix!

**Pull request merged**